models/classes/export/AbstractTestExport.php

```diff
@@ -151,6 +141,16 @@ public function export($formValues, $destination): Report
         return $report;
     }
 
+    protected function fireTestExportEvent(Resource $test)
```

Comment:
`trigger` is a more expected term, but `fire` works as well :)

Reply:
Renamed